src/dialogs/voice-assistant-setup/voice-assistant-setup-step-pipeline.ts

```diff
@@ -69,53 +69,89 @@ export class HaVoiceAssistantSetupStepPipeline extends LitElement {
     }
 
     return html`<div class="content">
-      <h1>What hardware do you want to use?</h1>
+      <h1>
+        ${this.hass.localize(
+          "ui.panel.config.voice_assistants.satellite_wizard.pipeline.title"
+        )}
+      </h1>
       <p class="secondary">
-        How quickly your assistant responds depends on the power of the
-        hardware.
+        ${this.hass.localize(
+          "ui.panel.config.voice_assistants.satellite_wizard.pipeline.secondary"
+        )}
       </p>
       <div class="container">
         <div class="messages-container cloud">
           <div class="message user ${this._showFirst ? "show" : ""}">
             ${!this._showFirst ? "…" : "Turn on the lights in the bedroom"}
           </div>
           ${this._showFirst
-            ? html`<div class="timing user">0.2 seconds</div>`
+            ? html`<div class="timing user">
+                0.2
+                ${this.hass.localize(
+                  "ui.panel.config.voice_assistants.satellite_wizard.pipeline.seconds"
+                )}
```

Comment:
We have a new duration formatting available that may be more suitable for all these.

Reply:
Problem is that it only supports integers and not float values :-/